signalfx-java/src/main/java/com/signalfx/signalflow/WebSocketTransport.java

```diff
@@ -192,6 +191,7 @@ public void keepalive(String handle) {
         private int timeout = DEFAULT_TIMEOUT;
         private int version = 2;
         private boolean compress = true;
+        private int maxBinaryMessageSize = -1;
```

Comment:
What's the impact of using -1 as the default?

Reply:
Hi @mpetazzoni, the current implementation is already using -1 as default parameter.

The Jetty WebSocketClient that is being used by SignalFx has -1 as default:

https://github.com/eclipse/jetty.project/blob/jetty-8.1.x/jetty-websocket/src/main/java/org/eclipse/jetty/websocket/WebSocketClient.java#L90